docs/tutorial.md

```diff
@@ -49,17 +49,20 @@ graph LR;
     D11[DMFT at T<sub>1</sub>] --> E11([Output calculation P<sub>1</sub>, T<sub>1</sub>])
     D12[DMFT at T<sub>2</sub>] --> E12([Output calculation P<sub>1</sub>, T<sub>2</sub>])
 ```
-Here "Input structure" refers to the _input_ material on which the calculation is performed, "DFT", "TB" and "DMFT" each of the _tasks_ of our calculation, and "Output calculation" the _output_ calculation of each of the final DMFT tasks.
+Here, "Input structure" refers to the all _input_ material given to perform the calculation (e.g., atomic structure, model, parameters, etc.). "DFT", "TB" and "DMFT" refer to individual _tasks_ of the workflow, which each correspond to a _SinglePoint_ entry in NOMAD. "Output calculation" refers to the _output_ data of each of the final DMFT tasks.
 
-Each task is then parsed into what we call a _SinglePoint_ entry in NOMAD. Our goal is then to set up the following workflows:
+The goal of this tutorial is to set up the following workflows:
 
-1. An example for a `SinglePoint` workflow using the `pressure1` subfolder data. This file can be ignored for later steps in the Tutorial.
-1. A workflow entry for each pressure P<sub>i=1,2</sub> grouping all `SinglePoint` "DFT", "TB", "DMFT at T<sub>1</sub>", and "DMFT at T<sub>2</sub>" tasks.
-2. An extra workflow entry grouping all pressure and all temperature calculations.
+1. A `SinglePoint` workflow for the `pressure1` subfolder data.
```

Comment:
Replace by:

 A `SinglePoint` workflow for one of the calculations (e.g., the DFT one) in the `pressure1` subfolder.

Reply:
👍